run_local_with_dependencies.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-sm --start API_DEFINITION API_EXAMPLE_MICROSERVICE -f
+sm --start API_DOCUMENTATION SERVICE_LOCATOR API_DEFINITION API_EXAMPLE_MICROSERVICE -f
```

Comment:
Should this drop:
- SERVICE_LOCATOR
 - API_DOCUMENTATION

And have
 -  COMBINED_API_DEFINITION

Reply:
That's that flipping revert.

Reply:
Sorted